### ISSUE TITLE
Segfault on start

### ISSUE THREAD

**Issue by MisterY** [loki-project/session-desktop#911]
**Describe the bug**
The app crashes on start.

**To Reproduce**
execute the AppImage file.

**Screenshots or Logs**
```
Set Windows Application User Model ID (AUMID) { appUserModelId: 'org.whispersystems.session-messenger-desktop' }
NODE_ENV production
NODE_CONFIG_DIR /tmp/.mount_sessioaBe6iU/resources/app.asar/config
NODE_CONFIG {}
ALLOW_CONFIG_MUTATIONS undefined
HOSTNAME andromede
NODE_APP_INSTANCE undefined
SUPPRESS_NO_CONFIG_WARNING undefined
userData: /home/alen/.config/Session
config/get: Successfully read user config file
config/get: Did not find ephemeral config file, cache is now empty object
making app single instance
{"name":"log","hostname":"andromede","pid":13922,"level":30,"msg":"app ready","time":"2020-02-22T18:30:37.471Z","v":0}
{"name":"log","hostname":"andromede","pid":13922,"level":30,"msg":"starting version 1.0.2","time":"2020-02-22T18:30:37.472Z","v":0}
{"name":"log","hostname":"andromede","pid":13922,"level":30,"msg":"updateSchema: Current schema version: 0; Most recent schema version: 11; SQLite version: 3.20.1; SQLCipher version: 3.4.2;","time":"2020-02-22T18:30:37.492Z","v":0}
{"name":"log","hostname":"andromede","pid":13922,"level":30,"msg":"updateToSchemaVersion1: starting...","time":"2020-02-22T18:30:37.492Z","v":0}
Segmentation fault (core dumped)
```

**Other information (please complete the following information):**

* Device: PC
* OS: openSuse Tumbleweed
* Session Version or Git commit hash:
1.0.2

**Comment by KeeJef** [comment]
looks related to #875 , think this could be happening on tails too

**Comment by yazeed44** [comment]
The same thing happens to me too

**Comment by KeeJef** [comment]
Should be resolved in releases 1.0.3 and up
